proto/stream_plan.proto

```diff
@@ -650,7 +650,7 @@ message StreamFragmentGraph {
   // edges between fragments.
   repeated StreamFragmentEdge edges = 2;
 
-  repeated uint32 dependent_table_ids = 3;
+  repeated uint32 dependent_relation_ids = 3;
```

Comment:
Actually renaming a field in proto message doesn't affect backward compatibility. 🥵 
![image](https://user-images.githubusercontent.com/14160025/220036489-a0a479f0-6fb3-4eb8-a566-c34c3d96ab15.png)

Shall we reserved the sequence and add a new one to pass it? Or simply just leave the field unchanged with some comments? @BugenZhao wdyt?

Reply:
This is because of the `WIRE_JSON` rule I set for the linter. 🥵 The motivation is that the names of some fields are depended by some components, like human-readable encoding for system params, and the code of dashboard. We can ignore this as it should be a soft failure.